worldengine/common.py

```diff
@@ -80,32 +81,31 @@ def rescale_value(original, prev_min, prev_max, min, max):
     return min + ((max - min) * f)
 
 
-def anti_alias(elevation, steps):
+def anti_alias(map, steps):#TODO: There is probably a bit of numpy-optimization that can be done here.
```

Comment:
I spent an hour trying to optimize this, making heavy use of numpy. For some reason that ended up even slower, though. Somebody else could try himself, since this method is one of the last few that overall take a huge amount of processing time (20% of the toal).